cli/src/common/imagebuilder_utils.py

```diff
@@ -20,6 +20,32 @@
 PCLUSTER_RESERVED_VOLUME_SIZE = 15
 AMI_NAME_REQUIRED_SUBSTRING = " {{ imagebuilder:buildDate }}"
 InstanceRole = Enum("InstanceRole", ("ROLE", "INSTANCE_PROFILE"))
+ImageBuilderStatusMapping = {
```

Comment:
Why do you need them in the utils?
Let's avoid the "utils" module if possible, I'm also trying to cleanup our old utility module by moving the methods in the right layer/module.

If they are constant lets move to imagebuilder_constants module.
if they are just used by the model we can have this definition there.

Reply:
Moved mapping to imagebuilder module, should we drop utils? Is there any benefit for doing that?

Reply:
Yes, I think we should get rid of the utils module or try to move most of the things in the appropriate module.
See some interesting discussions about this:
https://stackoverflow.com/questions/3340032/utility-classes-are-evil
https://breadcrumbscollector.tech/stop-naming-your-python-modules-utils/